model_analyzer/config/generate/quick_plus_concurrency_sweep_run_config_generator.py

```diff
@@ -115,90 +117,23 @@ def _create_quick_run_config_generator(self) -> QuickRunConfigGenerator:
             client=self._client,
             model_variant_name_manager=self._model_variant_name_manager)
 
-    def _sweep_concurrency_over_top_results(self) -> Generator[RunConfig, None, None]:
+    def _sweep_concurrency_over_top_results(
+            self) -> Generator[RunConfig, None, None]:
         top_results = self._result_manager.top_n_results(
-            n=self._config.num_configs_per_model)
+            model_name=self._result_manager.get_model_names()[0],
```

Comment:
what is the [0] here?

Reply:
Good catch. I've fixed this and replaced it with a loop so that we would sweep over all models, if the user does a quick, non-concurrent search.